.github/workflows/patchTemplate.yml

```diff
@@ -1,4 +1,4 @@
-name: Patch Template
+name: Patch Scaffolds
```

Comment:
rename the file `patchTemplate.yml` to `patchScaffold.yml`?

Reply:
Good call.